suricata/update/net.py

```diff
@@ -91,7 +91,8 @@ def is_header_clean(header):
     if len(header) != 2:
         return False
     name, val = header[0].strip(), header[1].strip()
-    if re.match( r"^[\w-]+$", name) and re.match(r"^[\w-]+$", val):
+    if (re.match( r"^[\w-]+$", name) and
+        (re.match(r"^[\w-]+$", val) or re.match(r"^Basic\s[\w-]+$", val))):
```

Comment:
Should we not just allow white space in values?  I don't think that would make it invalid.

Reply:
Thanks for your reply. I agree with you. Solution v1 would be the better one.

See: https://github.com/OISF/suricata-update/pull/272

Reply:
ah, nice, didn't see that one.